.travis.yml

```diff
@@ -8,14 +8,13 @@ services:
 before_install:
 - sudo apt-get -qq update
 - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+- sudo apt-get install -y ruby bundler rake
```

Comment:
I wanted to call out the ruby "crazyness" on it's own line.  Hopefully we can remove that stuff at some point.